### DISCUSSION TITLE
Chinese-LLaMA-2-7B 指令模型需要什么配置

### DISCUSSION THREAD

**Discussion by Goddarkness** [ymcui/Chinese-LLaMA-Alpaca-2#372] [Q&A]
我用[Web图形界面交互形式](https://github.com/ymcui/Chinese-LLaMA-Alpaca-2/wiki/inference_with_transformers_zh#web%E5%9B%BE%E5%BD%A2%E7%95%8C%E9%9D%A2%E4%BA%A4%E4%BA%92%E5%BD%A2%E5%BC%8F)方式在Windows上部署，遇到error：**ValueError: The current `device_map` had weights offloaded to the disk. Please provide an `offload_folder` for them. Alternatively, make sure you have `safetensors` installed if the model you are using offers the weights in this format.** 是我配置不行吗？

**Comment by ovo-Tim** [comment]
可能是内存不足？你应该是在 cpu 上跑的，我也遇到过这个问题
只需要在对应代码，提供 `offload_folder` 参数就行了(应该是磁盘缓存？)
参照https://github.com/nomic-ai/gpt4all/issues/239
